src/plugins/persistence/couch/setup-couchdb.sh

```diff
@@ -123,3 +131,7 @@ if [ "{\"ok\":true}" == "${response}" ]; then
 else
     echo "Database permissions not updated"
 fi
+
+echo "Enabling CORS"
```

Comment:
this script is meant to be idempotent. What happens when this script is run twice?